pkgs/tools/misc/zsjed/default.nix

```diff
@@ -0,0 +1,62 @@
+{ coreutils
+, fetchurl
+, findutils
+, gawk
+, gnused
+, lib
+, stdenv
+}:
+
+stdenv.mkDerivation rec {
+  pname = "zsjed";
+  version = "0.3";
+
+  src = fetchurl {
+    url = "https://ytrizja.de/distfiles/zsjed-${version}.tar.gz";
+    sha256 = "1rcrvrqjsv80ldz7cscqbrf98jb0yd2in97irrzlpask699z5k29";
+  };
+
+  nativeBuildInputs = [
+    coreutils # mkdir, install
+    gnused
+  ];
```

Comment:
```suggestion
```
Should be included in stdenv.

Reply:
Is `diffutils` included in stdenv?

Reply:
I assume for now that is it not in stdenv.